invenio_app_rdm/theme/assets/semantic-ui/translations/invenio_app_rdm/package.json

```diff
@@ -9,10 +9,11 @@
   },
   "devDependencies": {
     "i18next-conv": "^10.2.0",
-    "i18next-scanner": "^3.0.0"
+    "i18next-scanner": "^3.0.0",
+    "react-i18next": "^11.11.3"
   },
   "scripts": {
-    "extract_messages": "i18next-scanner --config i18next-scanner.config.js '../../**/*.{js,jsx}'",
+    "extract_messages": "i18next-scanner --config i18next-scanner.config.js '../../js/**/*.{js,jsx}'",
```

Comment:
I have restricted it to only **js** folder - since it was looking for other files in `node_modules`.